test/test_builder.py

```diff
@@ -152,11 +157,38 @@ def test_table(self):
             self.assertEqual(lines[3], '|False|False|False|\n')
             self.assertEqual(lines[4], '|True|False|True|\n')
 
+    def test_no_parent(self):
+        self.assertEqual(ConfluenceDocMap.parent("toctree"), None)
+        self.assertEqual(ConfluenceDocMap.parent("code"), None)
+
     def test_publish(self):
         builder = ConfluenceBuilder(self.app)
         builder.config.confluence_publish = True
         with self.assertRaises(ConfluenceConfigurationError):
             builder.init()
 
+class TestConfluenceBuilderExperimental(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        basedir = os.path.dirname(os.path.realpath(__file__))
+        srcdir = os.path.join(basedir, 'testproj')
+        self.expected = os.path.join(srcdir, 'expected')
+        builddir = os.path.join(srcdir, 'build')
+        self.outdir = os.path.join(builddir, 'out')
+        doctreedir = os.path.join(builddir, 'doctree')
+
+        self.config = create_default_test_config()
+        self.config['master_doc'] = "toctree"
```

Comment:
this causes the following warning to display. We don't want to include contents.rst in any toctree because that would muck up other tests, so I don't know what to do about this. Test passes though. 

`/Users/johnteasdale/Uber/sphinxcontrib-confluencebuilder/test/testproj/contents.rst: WARNING: document isn't included in any toctree`